optaplanner-core/src/test/java/org/optaplanner/core/api/score/stream/uni/UniConstraintStreamTest.java

```diff
@@ -60,6 +60,7 @@ public UniConstraintStreamTest(boolean constraintMatchEnabled, ConstraintStreamI
 
     @Test
     public void filter_problemFact() {
+        assumeBavet();
```

Comment:
`@Before ... setUp() { assumeBavet(); }` isn't suitable for this class?

Reply:
Unfortunately, it is not. In this class, some tests already support both implementations.